assignments/closure.js

```diff
@@ -1,21 +1,64 @@
 // ==== Challenge 1: Write your own closure ====
 // Write a simple closure of your own creation.  Keep it simple!
 
+function sayHello() {
+  let greet = "Hello";
 
-/* STRETCH PROBLEMS, Do not attempt until you have completed all previous tasks for today's project files */
+  function sayWorld() {
+    console.log(greet, "World!");
+  }
+  return sayWorld;
+}
+let hi = sayHello();
+hi();
```

Comment:
```suggestion
console.log(hi());
```
And now you can see this working a little better